DB/projects.json

```
@@ -4779,5 +4779,15 @@
         ]
       }
     ]
-  }
-]
+  },
+  {
+  "github_username": "mohit-dhote",  
+  "Projects": [
+    {
+      "github_url": "https://github.com/mohit-dhote/mohitdhote.github.io",
+      "title": "my_portfolio",
+      "description": "A clean, beautiful, responsive portfolio template for Software Developers!",
+      "tech": ["html", "javascript", "CSS"]
```

convert to smallcase
```suggestion
      "tech": ["html", "javascript", "css"]
```